pkg/detector/detector.go

```diff
@@ -909,12 +909,19 @@ func (d *ResourceDetector) HandlePropagationPolicyDeletion(policyNS string, poli
 		return err
 	}
 
-	for _, binding := range rbs.Items {
+	for index, binding := range rbs.Items {
+		// Cleanup the labels from the reference binding so that the karmada scheduler won't reschedule the binding.
```

Comment:
The `karmada-scheduler` will prevent the binding without the PropagationPolicy label from the queue?

Reply:
Can you tell me where is the relevant code?

Reply:
> Can you tell me where is the relevant code?

Yes, I add this filter when implementing dependencies distributor as bindings created by dependencies distributor don't have this label. Code [here](https://github.com/karmada-io/karmada/blob/e85d376bcc3f545619cedd3549d5a7ee274ba74d/pkg/scheduler/event_handler.go#L74)

Reply:
I see it.
https://github.com/karmada-io/karmada/blob/e85d376bcc3f545619cedd3549d5a7ee274ba74d/pkg/scheduler/event_handler.go#L68-L76